backend/endpoints/files.py

```diff
@@ -0,0 +1,98 @@
+from fastapi import  HTTPException, status
+from schema.response import ResponseModel
+from starlette.responses import JSONResponse, Response
+from fastapi import FastAPI, UploadFile, File, APIRouter
+from fastapi.responses import FileResponse, JSONResponse
+from utils.message_utils import get_message, get_room_messages
+import os
+import zipfile
+import io
+from zipfile import ZipFile
+router = APIRouter()
+
+"""
+ A get method endpoint to Handle retrieving and Viewing of uploaded files in a DM
+
+"""
```

Comment:
what is this for? did you miss some codes?

Reply:
no